tests/views/resource_tests.py

```diff
@@ -54,6 +65,19 @@ def setUpTestData(cls):
             edit.userid = user.id
             edit.save()
 
+        with open(
+            os.path.join("tests/fixtures/jsonld_base/models/4564-referenced.json"),
+            "r",
+        ) as f:
+            archesfile = JSONDeserializer().deserialize(f)
+        ResourceGraphImporter(archesfile["graph"])
+        with open(
+            os.path.join("tests/fixtures/jsonld_base/models/4564-person.json"),
+            "r",
+        ) as f:
+            archesfile = JSONDeserializer().deserialize(f)
+        ResourceGraphImporter(archesfile["graph"])
+
```

Comment:
Hey @whatisgalen. In 7.6 we added a class attribute `graph_fixtures` to ArchesTestCase for this, so we may as well start using it more. It doesn't seem to make a difference with these new tests, but I think with others we saw subtle tricky things like having to orchestrate ontology loads and syncing language settings and stuff like that. If we go through the central interface then each test doesn't need to rediscover that.

```diff
diff --git a/tests/test_settings.py b/tests/test_settings.py
index 78982437c8..1aa77d1146 100644
--- a/tests/test_settings.py
+++ b/tests/test_settings.py
@@ -44,6 +44,12 @@ RESOURCE_GRAPH_LOCATIONS = [
         "graphs",
         "resource_models",
     ),
+    os.path.join(
+        TEST_ROOT,
+        "fixtures",
+        "jsonld_base",
+        "models",
+    ),
 ]
 REFERENCE_DATA_FIXTURE_LOCATION = os.path.join(
     TEST_ROOT, "fixtures", "testing_prj", "testing_prj", "pkg", "reference_data"
diff --git a/tests/views/resource_tests.py b/tests/views/resource_tests.py
index 929b5f1504..68507887da 100644
--- a/tests/views/resource_tests.py
+++ b/tests/views/resource_tests.py
@@ -16,7 +16,6 @@ You should have received a copy of the GNU Affero General Public License
 along with this program. If not, see <http://www.gnu.org/licenses/>.
 """
 
-import os
 from unittest.mock import MagicMock, patch
 from arches.app.views.resource import ResourcePermissionDataView
 from tests.base_test import ArchesTestCase
@@ -26,10 +25,7 @@ from arches.app.models.resource import Resource
 from arches.app.models.tile import Tile
 from tests.utils.search_test_utils import sync_es
 from arches.app.search.search_engine_factory import SearchEngineFactory
-from arches.app.utils.betterJSONSerializer import JSONSerializer, JSONDeserializer
-from arches.app.utils.data_management.resource_graphs.importer import (
-    import_graph as ResourceGraphImporter,
-)
+from arches.app.utils.betterJSONSerializer import JSONSerializer
 from django.contrib.auth.models import User
 from django.contrib.auth.models import Group
 from guardian.shortcuts import (
@@ -44,7 +40,7 @@ from tests.utils.permission_test_utils import add_users
 
 
 class CommandLineTests(ArchesTestCase):
-    graph_fixtures = ["Data_Type_Model"]
+    graph_fixtures = ["Data_Type_Model", "4564-referenced", "4564-person"]
     data_type_graphid = "330802c5-95bd-11e8-b7ac-acde48001122"
     resource_instance_id = "f562c2fa-48d3-4798-a723-10209806c068"
     reference_graphid = "e3d4505e-bfa7-11e9-b4dc-0242ac160002"
@@ -65,19 +61,6 @@ class CommandLineTests(ArchesTestCase):
             edit.userid = user.id
             edit.save()
 
-        with open(
-            os.path.join("tests/fixtures/jsonld_base/models/4564-referenced.json"),
-            "r",
-        ) as f:
-            archesfile = JSONDeserializer().deserialize(f)
-        ResourceGraphImporter(archesfile["graph"])
-        with open(
-            os.path.join("tests/fixtures/jsonld_base/models/4564-person.json"),
-            "r",
-        ) as f:
-            archesfile = JSONDeserializer().deserialize(f)
-        ResourceGraphImporter(archesfile["graph"])
-
     def test_resource_instance_permission_assignment(self):
         """
         Test that we can assign resource instance permissions
```

Reply:
@jacobtylerwalls yea I noticed that class property however it seems to only look in one particular directory. Out of convenience I decided to use those particular models since one had a resource-instance node, however they're nested in that subdir `jsonld_base/models/` which was not searched by the logic using `graph_fixtures`